morula.yml

```diff
@@ -1,4 +1,4 @@
-before-all: ""
+before-all: "exorelay-javascript"
```

Comment:
This is required to be built so that `exoservice-javascript` has access to it for building the webpack bundle.